src/08_comprehensions.py

```diff
@@ -2,40 +2,45 @@
 List comprehensions are one cool and unique feature of Python.
 They essentially act as a terse and concise way of initializing
 and populating a list given some expression that specifies how
-the list should be populated. 
+the list should be populated.
 
 Take a look at https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions
 for more info regarding list comprehensions.
 """
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [num for num in range(6) if num > 0]
+print (y)
+
 
+y = [ i + 1 for i in range(5)]
 print (y)
 
+
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
-y = []
-
+y = [pow(num, 3) for num in range(10)]
```

Comment:
Looks good, exponent operator works here as well. `[num**3 for num in range(10)]`